kolibri/core/public/api.py

```diff
@@ -324,3 +326,16 @@ def create(self, request):
 
     def update(self, request, pk=None):
         return self.check_queue(request, pk=pk)
+
+
+class FacilitySearchUsernameViewSet(ReadOnlyValuesViewset):
```

Comment:
This is a spec issue rather than an implementation issue - but I think we might want to revisit how much information about users we let be publicly accessible.

Reply:
Changed to return just if the username exists when password is needed to login. If not, it does an actual search and returns matching users with their ids